java/client/src/main/java/glide/api/RedisClusterClient.java

```diff
@@ -118,4 +122,40 @@ public CompletableFuture<ClusterValue<String>> info(
         return commandManager.submitNewCommand(
                 Info, options.toArgs(), route, response -> ClusterValue.of(handleObjectResponse(response)));
     }
+
+    /** {@inheritDoc} The command will be routed a random node. */
```

Comment:
Do we want to follow this throughout - whenever we have a command with an extra comment when used in clustermode?  

Problem I have is it breaks convention and we now have documentation in the implementation files.

Reply:
Fixed in cf519a0c